docker/utils/update_config_directory.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+# method to copy directory/file using package name
+copy_package_file() {
+  local package_name=$1
+  local file_name=$2
+  local package_file_path=$(ros2 pkg prefix $package_name)/share/$package_name/$file_name
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)